conans/client/cache/remote_registry.py

```diff
@@ -9,7 +10,8 @@
 from conans.model.ref import PackageReference, ConanFileReference
 
 
-Remote = namedtuple("Remote", "name url verify_ssl")
+Remote = namedtuple("Remote", "name url verify_ssl disabled")
+Remote.__new__.__defaults__ = (None, None, None, False)
```

Comment:
Is this needed? I think it is better to just provide the `False` value when a tuple instance is created.